app/api/schema/access_codes.py

```diff
@@ -33,10 +33,9 @@ def validate_date(self, data, original_data):
             if 'valid_from' not in data:
                 data['valid_from'] = access_code.valid_from
 
-            if 'valid_till' not in data:
-                data['valid_till'] = access_code.valid_till
+            data['valid_till'] = access_code.valid_expire_time
```

Comment:
Defeats the purpose as previously reviewed. Now the time is static. We don't want to store the end time if when sales ends at is selected.

Ask in a call if you are still confused